src/styles/block/card.scss

```diff
@@ -0,0 +1,101 @@
+.card {
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
+  width: $product-card-width;
+  background-color: #fff;
+  border: 1px solid $color-gray;
```

Comment:
The variable `$color-gray` is used here. Ensure that this variable is defined in your SCSS files. If it's not defined, it will cause a compilation error.